index.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     'declaration-property-value-disallowed-list': [
       { '/^border/': ['none'] },
       { severity: 'error' },
+      { 'text-transform': 'uppercase' },
```

Comment:
I think this line can be removed now, as we are disallowing the property

Reply:
done